pkgs/development/libraries/slang/default.nix

```diff
@@ -1,10 +1,19 @@
-{ stdenv, fetchurl, ncurses, pcre, libpng, zlib, readline, libiconv }:
+{ stdenv, fetchurl
+, libiconv
+, libpng
+, ncurses
+, pcre
+, readline
+, zlib
+}:
 
 stdenv.mkDerivation rec {
-  name = "slang-2.3.2";
+  pname = "slang";
+  version = "2.3.2";
+
   src = fetchurl {
-    url = "https://www.jedsoft.org/releases/slang/${name}.tar.bz2";
-    sha256 = "06p379fqn6w38rdpqi98irxi2bf4llb0rja3dlgkqz7nqh7kp7pw";
+    url = "https://www.jedsoft.org/releases/slang/${pname}-${version}.tar.bz2";
+    sha256 = "sha256-/J47D8T2fDwfbUPJDBalxC0Re44oRXxbRoMbi1064xo=";
```

Comment:
This commit is messy; it is mixing your addition to the maintainers list and code rework. Please split it.

Reply:
It is not so messy, I think. It is just a batch update, following a logical flow from dependencies to apps.

Reply:
Please add to the commit something like ``, cleanups`` or similar.

Reply:
Well, your commit log says otherwise. It says you added yourself as maintainer but the diff does way more than that.

Reply:
Oh, I understand now. I will fix this.